docs/src/main/sphinx/connector/hive.rst

```diff
@@ -164,7 +164,7 @@ functionality:
 Configuration
 -------------
 
-Create ``etc/catalog/hive.properties`` with the following contents
+Create ``/etc/trino/catalog/hive.properties`` with the following contents
```

Comment:
The new path is helpful for docker users, but it's not helpful for non-docker users and changing only Hive documentation doesn't make sense. 

@mosabua I remember there was similar comment previously. Do you have any idea to improve our documentation?

Reply:
I think the confusion comes from two things:

- it's not clear that `etc/catalog/hive.properties` is supposed to be in the installation directory
- it's not obvious (to me at least) how to find the installation directory. Docker users have to find it by basically digging through the Dockerfile, reading the `run-trino` and `launch.py` scripts.

So maybe how bout something like this?

```
Create a hive.properties file with the following contents:

...

Place the file in $INSTALL_DIR/catalog/hive.properties

For Docker users, the $INSTALL_DIR is `/etc/trino` as defined [here](https://github.com/trinodb/trino/blob/master/core/docker/README.md#configuration)

For non-Docker users, the $INSTALL_DIR is the location of your Trino server tarball  as defined [here](https://trino.io/docs/current/installation/deployment.html#installing-trino).
```

I don't know if this is too verbose, but it was a point of confusion for me as a new user 🤷

Reply:
The deployment doc is for the tarball. It has to be broken up and we have to create a separate Docker one ... its on our backlog to get to. For now I suggest to leave the docs as they are since everything is based on the tarball conventions ...